<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="EPRURYAW0075" name="com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest" tests="6" failures="0" timestamp="27 Sep 2016 13:47:24 GMT" time="73.868" errors="5">
  <testcase name="sendMessagesTest" time="20.058" classname="com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest">
    <error type="org.openqa.selenium.TimeoutException" message="Timed out after 15 seconds waiting for com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage$1@58ce9668
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;EPRURYAW0075&#039;, ip: &#039;192.168.56.1&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_92&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 15 seconds waiting for com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage$1@58ce9668
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'EPRURYAW0075', ip: '192.168.56.1', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_92'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:292)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:261)
	at com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage.fluentWaitMethod(MessagePage.java:124)
	at com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage.sendMessage(MessagePage.java:98)
	at com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest.sendMessagesTest(YandexMailTest.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- sendMessagesTest -->
  <testcase name="deleteMessage" time="0.000" classname="com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest">
    <error type="java.lang.Throwable" message="Method YandexMailTest.deleteMessage(java.lang.String)[pri:0, instance:com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest@352ff4da] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method YandexMailTest.deleteMessage(java.lang.String)[pri:0, instance:com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest@352ff4da] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1033)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- deleteMessage -->
  <testcase name="sendMessagesTest" time="19.229" classname="com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest">
    <error type="org.openqa.selenium.TimeoutException" message="Timed out after 15 seconds waiting for com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage$1@7efaad5a
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;EPRURYAW0075&#039;, ip: &#039;192.168.56.1&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_92&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 15 seconds waiting for com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage$1@7efaad5a
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'EPRURYAW0075', ip: '192.168.56.1', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_92'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:292)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:261)
	at com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage.fluentWaitMethod(MessagePage.java:124)
	at com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage.sendMessage(MessagePage.java:98)
	at com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest.sendMessagesTest(YandexMailTest.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- sendMessagesTest -->
  <testcase name="loginTest" time="4.841" classname="com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest"/>
  <testcase name="getException" time="0.000" classname="com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest">
    <error type="java.lang.Throwable" message="Method YandexMailTest.getException()[pri:0, instance:com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest@352ff4da] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method YandexMailTest.getException()[pri:0, instance:com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest@352ff4da] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1033)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- getException -->
  <testcase name="sendMessagesTest" time="29.740" classname="com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest">
    <error type="org.openqa.selenium.TimeoutException" message="Timed out after 15 seconds waiting for com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage$1@5ddcc487
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:37:03&#039;
System info: host: &#039;EPRURYAW0075&#039;, ip: &#039;192.168.56.1&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_92&#039;
Driver info: driver.version: unknown">
      <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 15 seconds waiting for com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage$1@5ddcc487
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'EPRURYAW0075', ip: '192.168.56.1', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_92'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.FluentWait.timeoutException(FluentWait.java:292)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:261)
	at com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage.fluentWaitMethod(MessagePage.java:124)
	at com.epam.vyacheslav_utenkov.java.lesson7.ui.MessagePage.sendMessage(MessagePage.java:98)
	at com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest.sendMessagesTest(YandexMailTest.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </error>
  </testcase> <!-- sendMessagesTest -->
</testsuite> <!-- com.epam.vyacheslav_utenkov.java.lesson7.advanced.YandexMailTest -->
